ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh

```diff
@@ -62,8 +104,9 @@ else
     return 0
 fi
 
+# vmc.mirror-registry.qe.devcluster.openshift.com:5000
 MIRROR_PROXY_REGISTRY=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
-# get registry host name, no port
-REGISTRY_HOST=`echo ${MIRROR_PROXY_REGISTRY} | cut -d \: -f 1`
+set_CA_for_nodes
+config_mirror_auth
 mirror_tag_images
```

Comment:
@xiuwang do you know if sample co get healthy only mirror these images, but not creating icsp for 6001 and 6002 mirror registry?

Reply:
Correct, samples co will not depend on icsp, the openshift imagesteams point to mirror registry images with tag not used the icsp.